app.rb

```diff
@@ -1,8 +1,19 @@
-require 'sinatra/base'
+require "sinatra/base"
+require "sinatra/reloader"
+require "./lib/peep"
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  # configure :development do
+  #   register Sinatra::Reloader
+  # end
+
+  get "/test" do
+    "Test page"
+  end
+
+  get "/peeps" do
```

Comment:
Nice and concise - easy to follow the train of thought and the route that the code will follow